cura/PrinterOutput/PrintJobOutputModel.py

```diff
@@ -91,7 +91,7 @@ def updateOwner(self, owner):
     def assignedPrinter(self):
         return self._assigned_printer
 
-    def updateAssignedPrinter(self, assigned_printer: "PrinterOutputModel"):
+    def updateAssignedPrinter(self, assigned_printer: Optional["PrinterOutputModel"]):
```

Comment:
More nitpicking; Return type should also be None.

Reply:
Fixed in a36deea